db/migrate/20211209022316_create_tracks.rb

```diff
@@ -1,20 +0,0 @@
-class CreateTracks < ActiveRecord::Migration[6.1]
```

Comment:
You cannot just delete migrations, it breaks DB consistency when. If you want to remove table create new migration with deletion of table